### ISSUE TITLE
Installation error on Mac: `No module named 'future' `

### ISSUE THREAD

**Issue by dokterbob** [rduivenvoorde/nominatim_locator_filter#3]
```
Kon plug-in 'nominatim_locator_filter' niet laden wegens een fout bij het aanroepen van de methode classFactory() 

ModuleNotFoundError: No module named 'future' 
Traceback (most recent call last):
  File "/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 335, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/nominatim_locator_filter/__init__.py", line 10, in classFactory
    from .nominatimfilter import NominatimFilterPlugin
  File "/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 672, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/nominatim_locator_filter/nominatimfilter.py", line 6, in 
    from . networkaccessmanager import NetworkAccessManager, RequestsException
  File "/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 672, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/nominatim_locator_filter/networkaccessmanager.py", line 21, in 
    from future import standard_library
  File "/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 672, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'future'


Python versie: 3.6.8 (v3.6.8:3c6b436a57, Dec 24 2018, 02:04:31) [GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] 
QGIS versie: 3.6.2-Noosa Noosa, exported 

Python pad:
/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python
/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python
/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins
/Applications/QGIS3.6.app/Contents/MacOS/../Resources/python/plugins
/Library/Frameworks/SQLite3.framework/Versions/E/Python/3.6
/Library/Frameworks/Python.framework/Versions/3.6/lib/python36.zip
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/lib-dynload
/Users/drbob/Library/Python/3.6/lib/python/site-packages
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
/Users/drbob/Library/Application Support/QGIS/QGIS3/profiles/default/python
/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/IPython/extensions
```

**Comment by rduivenvoorde** [comment]
@dokterbob is this still an issue... in one of the 'helping' modules apparently 'future' is still imported, but for what I know not needed anymore for Pyhon3...
So I is easy to remove actually
(sorry for this late response, github issues are bounced pretty often with my email)

**Comment by rduivenvoorde** [comment]
confirmed here on new Fedora.. so YES future should be removed!

**Comment by rduivenvoorde** [comment]
should be fixed in 0.2.4
